app/src/main/java/com/example/faunaapp/view/fragments/LogInFragment.java

```diff
@@ -0,0 +1,57 @@
+package com.example.faunaapp.view.fragments;
+
+import android.os.Bundle;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.lifecycle.ViewModelProvider;
+
+import com.example.faunaapp.R;
+import com.example.faunaapp.view_model.LogInFragmentViewModel;
+import com.google.android.material.textfield.TextInputLayout;
+
+import java.util.Objects;
+
+public class LogInFragment extends Fragment {
+
+    private Button logInButton;
+    private TextInputLayout email, password;
+    private LogInFragmentViewModel logInViewModel;
+    private View logInView;
```

Comment:
Are you actively using the private View? If not do not declare it.

Reply:
I do use logInView in methods : onCreateView, initializeFragmentsValue, and in setSnackbar